Assets/Python/OOSLogger.py

```
@@ -1,190 +1,453 @@
+            # Space at end of player's info
+            w("\n\n")
+
+            # Intelligent garbage collection based on memory pressure
+            if hybrid_gc.should_collect(iPlayer):
+                collected = hybrid_gc.collect(iPlayer)
+                # Keep file open; just flush to bound buffers
+                try:
+                    pFile.flush()
+                    try:
+                        os.fsync(pFile.fileno())
+                    except Exception, _e:
```

Ruff (0.12.2)

SyntaxError: Multiple exception types must be parenthesized on Python 3.13 (syntax was added in Python 3.14)

---

Undefined name `_e`

(F821)